NOTES-RELEASE.md

```diff
@@ -0,0 +1,13 @@
+
+###v0.1.0
```

Comment:
There is missing space between `###` and number version

Reply:
Done